src/participantDetails.js

```diff
@@ -104,11 +107,14 @@ export const render = (participant, changedOption) => {
             const shouldHideButton = !row.editable || hideLoginInformation;
             const disableButton = hideLoginInformation || !row.editable;
             const variableLabel = row.label;
-            const variableValue = participant[conceptId];
+            const variableValue = Object.prototype.hasOwnProperty.call(changedOption, conceptId) ? changedOption[conceptId] : participant[conceptId];
```

Comment:
I saw `Object.hasOwn()` is recommended over `Object.prototype.hasOwnProperty()`. We may consider the migration.